5.2/5.2.1/debian/Dockerfile

```diff
@@ -33,6 +33,10 @@ RUN buildDeps="dpkg-dev gcc libbz2-dev libc6-dev libffi-dev libjpeg62-turbo-dev
  && pip install pip==$PIP setuptools==$SETUPTOOLS zc.buildout==$ZC_BUILDOUT wheel==$WHEEL \
  && cd /plone/instance \
  && buildout \
+ && mkdir /plone/.buildout \
+ && echo "[buildout]" > /plone/.buildout/default.cfg \
+ && echo "download-cache = /plone/buildout-cache/downloads" >> /plone/.buildout/default.cfg \
+ && echo "eggs-directory = /plone/buildout-cache/eggs" >> /plone/.buildout/default.cfg \
```

Comment:
This shouldn't be needed as it's the default place where `downloads` and `eggs` are stored.